src/main/resources/META-INF/reaktivity/socks.idl

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2016-2017 The Reaktivity Project
+ *
+ * The Reaktivity Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+scope socks
+{
+    enum SocksMode
```

Comment:
I don't think Mode will be needed.  As this PR only implements FORWARD tests, I would suggest removing Mode for now.  And adding it back if in the unlikely event it is needed in the future.